boards/xtensa/esp32/common/src/esp32_board_spi.c

```diff
@@ -84,6 +84,19 @@ static inline int spi_cmddata(struct spi_dev_s *dev, uint32_t devid,
     }
 #endif
 
+#ifdef CONFIG_LCD_SSD1680
```

Comment:
I do not see that `LCD_SSD1680` is supported in `spi_status`, maybe we can add it?